packages/block-library/src/buttons/edit.native.js

```diff
@@ -133,7 +140,18 @@ export default function ButtonsEdit( {
 			{ resizeObserver }
 			<InnerBlocks
 				allowedBlocks={ ALLOWED_BLOCKS }
-				template={ BUTTONS_TEMPLATE }
+				template={
+					preferredStyle
```

Comment:
Just make the quick update to `native` file as well and it will be good to go.

Reply:
Good catch, done :+1: